packages/application-shell/src/components/quick-access/create-commands.js

```diff
@@ -316,6 +316,7 @@ export default ({
         ].filter(Boolean),
       },
     project &&
+      project.languages &&
```

Comment:
Could we also use directly the conditional chain? something like `project?.languages?.length > 1 `

Reply:
I think we said we prefer not to use this syntax in the public repos

Reply:
OMG 🤦‍♂️ My bad, though we were in the mc repo 🙏